src/lib.rs

```diff
@@ -2,23 +2,23 @@ pub mod components;
 pub mod entities;
 mod metrics;
 pub mod middlewares;
-pub mod routes;
+pub mod api;
 mod utils;
 
 use actix_web::body::MessageBody;
 use actix_web::dev::{Server, ServiceFactory};
-use actix_web::middleware::{self};
+use actix_web::middleware;
```

Comment:
I think sometimes using `use module_name::{self}` and `use module_name` can have different effects. Are we sure this will work ok?

Reply:
In this case, the use of self was redundant as an example of: https://www.oreilly.com/library/view/learning-rust/9781785884306/fd7015f9-5c21-4551-8743-7b8d8dfd39bb.xhtml